src/main/java/com/google/api/codegen/config/GapicProductConfig.java

```diff
@@ -197,7 +197,14 @@ public static GapicProductConfig create(
     }
 
     // Toggle on/off proto annotations parsing.
-    ProtoParser protoParser = new ProtoParser(getDefaultLanguageFeatureConfig(language, null));
+    ProtoParser protoParser;
+    // TODO(andrealin): Expose command-line option for toggling proto annotations parsing.
+    if (configProto == null) {
+      // By default, enable proto annotations parsing when no GAPIC config is given.
+      protoParser = new ProtoParser(true);
+    } else {
+      protoParser = new ProtoParser(getDefaultLanguageFeatureConfig(language, null));
```

Comment:
Isn't the purpose of Proposal 2 from your doc that this should always be `false`? If so, can we just make it `false`? Or is there benefit to retaining the `getDefaultLanguageFeatureConfig` method?

Reply:
Hmm yes I don't think there will be any language-specific exemptions anytime soon.